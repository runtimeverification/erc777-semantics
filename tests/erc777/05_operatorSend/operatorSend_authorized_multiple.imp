test.preamble();

// authorize non-default, send, send

test.authorizeOperator(1);
test.makeCaller(1);
test.operatorSend(7, 2, 10, "", "");
test.operatorSend(7, 2, 10, "", "");
