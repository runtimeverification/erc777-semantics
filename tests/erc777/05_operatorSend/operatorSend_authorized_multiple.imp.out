7 is authorizing 1 as an operator
Changing the caller to 1
1 is sending 10 from 7 to 2 (data:""),  (operatorData:"")
Balance of 7 is 60
Balance of 2 is 30
1 is sending 10 from 7 to 2 (data:""),  (operatorData:"")
Balance of 7 is 50
Balance of 2 is 40
